src/components/tracks/apply/_index.tsx

```diff
@@ -7,8 +9,68 @@ interface IHeaderProps {
 
 // this component will be thrown away, so quick & dirty
 class Apply extends React.Component<IHeaderProps, {}> {
+  constructor() {
+    super();
```

Comment:
super in react accept an argument: props, call it like `super(props)` 

https://overreacted.io/why-do-we-write-super-props/